### PR TITLE
Fix UI in Settings in mobile view

### LINKED ISSUE #2086: Fix UI in Settings in mobile view
**Actual Behaviour**
Currently there is no padding and margin in chatapp, theme, etc.

**Expected Behaviour**
Add padding and margin and improve UI.

**Screenshots of the issue**
Screenshot 2019-05-13 at 10 23 52 PM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2086 
Changes: Added margin to save button in mobile view. And made "play demonstration" button left align in mobile view.
Demo Link: https://pr-2087-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-05-13 at 11 30 55 PM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Please review.

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes. Please review.

File: src/components/ChatApp/Settings/TextToSpeechSettings.react.js [hunk 1]
```diff
@@ -210,7 +212,7 @@ class TextToSpeechSettings extends Component {
             onClick={this.resetPitch}
           />
         </div>
-        <div style={{ textAlign: 'center' }}>
+        <div style={{ textAlign: MobileView <= 520 ? 'left' : 'center' }}>
```
Comment (akshatnitd) [inline src/components/ChatApp/Settings/TextToSpeechSettings.react.js]: MobileView return boolean, then what is this check for. Also keep camelCase standards for variable definition.

**Comment by harsh-jindal** [comment]
@akshatnitd Made the changes please review.

File: src/components/ChatApp/Settings/Settings.css [hunk 2]
```diff
@@ -219,4 +219,10 @@
   .reduceSettingDiv {
     width: 70%;
   }
+  .settingsSubmit {
+    margin: 1.25rem 0rem 1.25rem 1.875rem;
```
Comment (fragm3) [inline src/components/ChatApp/Settings/Settings.css]: Change this to: margin: 0 0 1.25rem 1.875rem;

Comment (fragm3) [inline src/components/ChatApp/Settings/Settings.css]: Change this to: margin: 0 0 1.25rem 1.875rem;

**Comment by harsh-jindal** [comment]
@fragm3 Done. Please review.

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd I have made some changes to incorporate for changes caused by upgrading to material ui core. Please review.

**Pull request merged**